src/main/java/eu/dissco/backend/service/AnnotationService.java

```diff
@@ -119,12 +119,18 @@ public AnnotationResponse persistAnnotation(AnnotationRequest annotation, String
 
   private AnnotationResponse mapResponseToAnnotationResponse(JsonNode response) {
     var annotation = response.get("annotation");
-    return new AnnotationResponse(response.get("id").asText(), response.get("version").asInt(),
-        annotation.get("type").asText(), annotation.get("motivation").asText(),
-        annotation.get("target"), annotation.get("body"), annotation.get("preferenceScore").asInt(),
+    return new AnnotationResponse(response.get("id").asText(),
+        response.get("version").asInt(),
+        annotation.get("type").asText(),
+        annotation.get("motivation").asText(),
+        annotation.get("target"),
+        annotation.get("body"),
+        annotation.get("preferenceScore").asInt(),
         annotation.get("creator").asText(),
-        Instant.ofEpochSecond(annotation.get("created").asLong()), annotation.get("generator"),
-        Instant.ofEpochSecond(annotation.get("generated").asLong()), null);
+        Instant.ofEpochSecond(annotation.get("created").asLong()),
+        annotation.get("generator"),
+        Instant.ofEpochSecond(annotation.get("generated").asLong()),
```

Comment:
Is the transfer from epoch to datetime already on the schedule or is it not relevant to this part?

Reply:
Yes it is on the backlog.